scripts/write-config.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
```

Comment:
When I have to read more than 6 lines of bash, my rate doubles 😄

Reply:
Tell me about it. I have to re-learn it every time I work on it. And then Codacy shows up and tells me I did it wrong :(